packages/appstore/src/components/wallet-header/wallet-currency-card.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { WalletIcon } from '@deriv/components';
+import { TWalletAccount } from 'Types';
+
+type TWalletCurrencyCard = Pick<TWalletAccount, 'is_demo' | 'currency' | 'icon'> & {
+    gradient_class?: string;
+    icon_type?: string;
+};
+
+const WalletCurrencyCard = ({ is_demo, currency, icon, icon_type, gradient_class }: TWalletCurrencyCard) => {
+    return (
+        <div className='wallet-header__currency' data-testid={`dt_${is_demo ? 'demo' : currency?.toLowerCase()}`}>
```

Comment:
according to the latest discussions about data-testid: it's better to add testid as an optional prop, and pass it from the test file. please do so or leave a TODO comment